### ISSUE TITLE
New customData key in the Protopus

### ISSUE THREAD

**Issue by scoiatael** [opendesigndev/octopus-specs#23]
In GitLab by @marekcerny on Feb 26, 2020, 14:52

We added to Avocode's Prototype Tool option to specify the origin/destination direction of the interaction. In other words, the user now can "create interaction from the **left side** of the layer Y to the **right side** of the artboard X". That's something that tools like XD, Figma, Sketch don't support.

To make this feature possible, we had to add some additional information to the Protopus.
Current Protopus structure:

```
artboards: []
fixedLayers: []
homeArtboards: []
interactions: {
  artboardOriginalId: [
    animation: {}
    bounds: {}
    destination: {}
    ownerId: ''
  ]
}
```

New Protopus structure:

```
artboards: []
fixedLayers: []
homeArtboards: []
interactions: {
  artboardOriginalId: [
    animation: {}
    bounds: {}
    destination: {}
    ownerId: string
    customData: {
      originDirection: number
      destinationDirection: number
    }
  ]
}
```

The new data are under the `customData` key.
If we need to add more data in the future, they will be under the same key.

**tl;dr** We just want to let you know that we added some custom data to the Protopus.
